osf/models/preprint.py

```diff
@@ -868,6 +868,7 @@ def set_privacy(self, permissions, auth=None, log=True, save=True, check_addons=
                 },
                 auth=auth,
                 save=False,
+                should_hide=True
```

Comment:
Also not sure if we want to hide these logs, it seems like intentional behavior, not sure about the use cases though.

Reply:
Let's also not hide these logs.